### PR TITLE
X-Rebirth Code Edition

### PR DESCRIPTION
**Pull request by leahcimNosliw**
- Replaced X-Rebirth picture with one that should now be small enough
- Added picture to the code
- Fixed title string
- Added releveant sub-trees (I think I have done this correctly now, but my Ruby is a little rusty. Sorry about that.)

### CODE REVIEW

File: config/initializers/constants.rb [hunk 1]
```diff
@@ -6025,5 +6025,234 @@ def new_vegas_specials
...
+					:'Free-play',
+					:'Aspiring Merchant',
+					:'Argon Mercenary',
+					:'Empire Builder',
+					:'The Teladi Outpost'
+				]
+			},
+			restrictions: {
+				title: 'Restrictions',
+				sub_trees: [primary_weapon]
```
Comment (Lockyy) [inline config/initializers/constants.rb]: This should be sub_trees: [:primary_weapon]

File: config/initializers/constants.rb [hunk 2]
```diff
@@ -6025,5 +6025,234 @@ def new_vegas_specials
...
+					:'Successfully board 1 ship without losing any marines',
+					:'Successfully board 20 ships without losing any marines',
+					:'Successfully board 100 ships without losing any marines',
+					:'Successfully board a Taranis, an Arawn and a Fulmekron',
+					:'Successfully board at least one of every possible ship'
+				]
+			},
+			build_goals: {
+				title: 'Build Goals',
+				sub_trees: [specific_zone_build],
```
Comment (Lockyy) [inline config/initializers/constants.rb]: Same as :primary_weapons above

**Comment by Lockyy** [comment]
Just those last two changes and this'll be ready to go.

**Comment by leahcimNosliw** [comment]
AH! My bad; I checked, I double-checked and I still missed them! I'll get on it in the morning :)

**Pull request closed without merging**